File: Python-Assignment/Chapter-Three/Question7.java
public class Question7{
   public static void main(String []args){

	System.out.printf("%10s%10s%10s%n", "Number", "Square", "Cube");

	for (int index = 0; index <= 5; index++){
		int indexSquared = index * index;
		int indexCubed = index * index * index;

		System.out.printf("%10d%10d%10d%n", index, indexSquared, indexCubed);
	}
   }
}

File: Python-Assignment/chapterThree/question7.py
print(f"{"Number":<10}{"Square":<10}{"Cube":<10}")

for index in range(0, 6):
	index_squared = index * index
	index_cubed = index * index * index

	print(f"{index:>6}{index_squared:>9}{index_cubed:>9}")